Controls/Cafe/Customer1.c

```diff
@@ -0,0 +1,29 @@
+#include "Cafe.h"
+
+/**
+ * Customer 1 wants to buy something!
+ */
+
+/**
+ * @brief Get the FOURTH item and cost on the menu (Cafe.h). Return the name and cost
+ * of the item to the pointers passed into the function. Add the cost to the business 
+ * revenue and print the current business revenue.
+ */
+void Customer1_checkout(char* name, int* cost) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
+/**
+ * @brief Acquire the register. Call checkout. Print the name and cost of the item.
+ * After checking out, release the register. Delay for FOUR seconds. Loop.
+ */
+void Task_Customer_1(void* p_arg) {
+	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
+	
+	// Use these pointers as parameters for Customer1_checkout
+	char* name;
+	int* cost;
```

Comment:
I'd make this an int, since that's really the value you'll be getting and want to store on the stack. That way, you can call Customer1_checkout(&name, &cost);

Reply:
Yeah I was being stupid